UniversalAlgebra/Varieties/EquationalLogic.lagda

```diff
@@ -1,4 +1,4 @@
----
+n---
```

Comment:
This leading `n` looks like a bug?

Reply:
That's definitely a bug... not sure how that crept in, but thanks for catching it!